dataflow/java/src/main/java/com/httparchive/dataflow/BigQueryImport.java

```diff
@@ -217,7 +217,8 @@ public void processElement(ProcessContext c) {
                         .set("payload", pageJSON);
                 c.output(pageRow);
 
-                if (lighthouse != null) {
+
+                if (lighthouse != null && lighthouse.isObject()) {
```

Comment:
The breakage happened because of a HAR file that had this:

```js
"_lighthouse":[]
```

When `lighthouse` is an array, the `for (JsonNode category : lighthouse.get("reportCategories"))` loop causes an exception. So ensuring the value is an object guards against the empty array case and prevents the exception.

Reply:
So the value is not null but some nonObject type? The wonder of Java..